src/api.js

```diff
@@ -5,16 +5,25 @@ import { loadMovie } from "./movies.js";
 import { getReviewsForMovieWithId } from "./reviewList.js";
 import loadScreenings from "./loadScreenings.js";
 import { filterScreenings } from "./filterScreenings.js";
+import { login, isAuthenticated } from "./auth.js";
 
 const apiRouter = express.Router();
 
+apiRouter.post("/token", (req, res) => { 
+	login(req, res);
```

Comment:
In my opinions, this delegation to `login()` is redundant, because the interface of `login` is identical to the function being passed to `apiRouter.post()`.

I would either move the content of `login()` here, or I would just pass `login` as the route hander, i.e. `apiRouter.post("/token", login)`.